services/provider/api/client/client.go

```diff
@@ -211,3 +212,66 @@ func (cc *OCSProviderClient) ReportStatus(ctx context.Context, consumerUUID stri
 
 	return cc.Client.ReportStatus(apiCtx, req)
 }
+
+func (cc *OCSProviderClient) OnboardStorageClusterPeer(ctx context.Context, onboardingTicket, storageClusterPeerUID string, remoteStorageCluster types.NamespacedName) (*pb.OnboardStorageClusterPeerResponse, error) {
+	if cc.Client == nil || cc.clientConn == nil {
+		return nil, fmt.Errorf("OCS client is closed")
+	}
+
+	req := &pb.OnboardStorageClusterPeerRequest{
+		OnboardingTicket:      onboardingTicket,
+		StorageClusterPeerUID: storageClusterPeerUID,
+		RemoteStorageCluster:  remoteStorageCluster.String(),
+	}
+
+	apiCtx, cancel := context.WithTimeout(ctx, cc.timeout)
+	defer cancel()
+
+	return cc.Client.OnboardStorageClusterPeer(apiCtx, req)
+}
+
+func (cc *OCSProviderClient) OffboardStorageClusterPeer(ctx context.Context, storageClusterPeerUID string) (*pb.OffboardStorageClusterPeerResponse, error) {
+	if cc.Client == nil || cc.clientConn == nil {
+		return nil, fmt.Errorf("OCS client is closed")
+	}
+
+	req := &pb.OffboardStorageClusterPeerRequest{
+		StorageClusterPeerUID: storageClusterPeerUID,
+	}
+
+	apiCtx, cancel := context.WithTimeout(ctx, cc.timeout)
+	defer cancel()
+
+	return cc.Client.OffboardStorageClusterPeer(apiCtx, req)
+}
+
+func (cc *OCSProviderClient) AcknowledgeOnboardingStorageClusterPeer(ctx context.Context, storageClusterPeerUID string) (*pb.AcknowledgeOnboardingStorageClusterPeerResponse, error) {
+	if cc.Client == nil || cc.clientConn == nil {
+		return nil, fmt.Errorf("provider client is closed")
+	}
+
+	req := &pb.AcknowledgeOnboardingStorageClusterPeerRequest{
+		StorageClusterPeerUID: storageClusterPeerUID,
+	}
+
+	apiCtx, cancel := context.WithTimeout(ctx, cc.timeout)
+	defer cancel()
+
+	return cc.Client.AcknowledgeOnboardingStorageClusterPeer(apiCtx, req)
+}
+
+func (cc *OCSProviderClient) GetMirroringInfo(ctx context.Context, storageClusterPeerUID string, blockPoolNames []string) (*pb.MirroringInfoResponse, error) {
```

Comment:
Shouldn't this be `GetPeerInfo`, onboarding peer does not have, conceptually, anything to do with mirroring. We might use it in the future to pass other type of information between 2 or more clusters.